{
  "file": "core/src/error.rs",
  "name": "<char::convert::CharTryFromError as error::Error>::description",
  "hash": "1502513335555468005712286196906760736382",
  "hash_direct": "96946748394414619425183418427363896532",
  "src": "fn description(&self) -> &str {\n        \"converted integer out of range for `char`\"\n    }"
}
